cms/src/components/content/EditContent.vue

```diff
@@ -184,6 +184,20 @@ const save = async () => {
     contentDocsPrev.value = _.cloneDeep(contentDocs.value);
 };
 
+const revertChanges = () => {
+    // Restore the parent document to the previous version
+    parent.value = _.cloneDeep(parentPrev.value!);
+
+    // Restore the content documents to the previous versions
+    contentDocs.value = _.cloneDeep(contentDocsPrev.value!);
+
+    addNotification({
+        title: `${capitaliseFirstLetter(titleType)} reverted`,
+        description: `The changes to the ${titleType} have been reverted`,
+        state: "info",
+    });
+};
+
```

Comment:
In this way, the notification will show even if there is changes. if you want to keep the notification, you can set differents messages according to the state of the content.

Reply:
Done.Good suggestion! It makes it much clearer to the user.